assets/js/modules/adsense/datastore/base.test.js

```diff
@@ -0,0 +1,61 @@
+/**
+ * `modules/adsense` base data store tests.
+ *
+ * Site Kit by Google, Copyright 2021 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * Internal dependencies
+ */
+import { createTestRegistry } from '../../../../../tests/js/utils';
+import { CORE_SITE } from '../../../googlesitekit/datastore/site/constants';
+import { MODULES_ADSENSE } from './constants';
+
+describe( 'modules/adsense base data store', () => {
+	let registry;
+	let store;
+
+	const adminURL = 'http://something.test/wp-admin';
+
+	beforeEach( () => {
+		jest.resetModules();
+
+		registry = createTestRegistry();
+		registry.dispatch( CORE_SITE ).receiveSiteInfo( {
+			adminURL,
+		} );
```

Comment:
Let's use the `provideSiteInfo` function instead of manually dispatching the `receiveSiteInfo` action here. Also, you can rely on the default value of the adminURL property that is defined there:

```js
beforeEach( () => {
    ...
    provideSiteInfo( registry );
}

it( 'does not define the admin page when unified dashboard is enabled', () => {
    ...
    expect( store.selectors.getAdminScreenURL() ).toBe(
        'http://example.com/wp-admin/admin.php?page=googlesitekit-dashboard'
    );
} );
```

The same applies to other tests.

Reply:
Thanks, that's useful to know - I've made the change. Cheers